doc/how_to_upgrade_persistent_instances.rst

```diff
@@ -1,520 +1,305 @@
-+----------------+-------------+-------------+
-| **keygen**     | t3a.small   | t3a.xlarge  |
-+----------------+-------------+-------------+
-| **distgit**    | t3a.medium  | t3a.medium  |
-+----------------+-------------+-------------+
-| **pulp**       | t3a.medium  | TODO        |
-+----------------+-------------+-------------+
-
-When more power is needed, please use the `ec2instances.info`_ comparator to get
-the cheapest available instance type according to our needs.
-
-
-5. Key pair (login)
-...................
-
-- Make sure to use existing key pair named ``Ansible Key``.  This allows us to
-  run the playbooks on ``batcave01`` box against the newly spawned VM.
-
-
-6. Network settings
-...................
+The goal is to complete as much pre-upgrade work as possible while focusing on
+minimizing the **outage window** and only performing essential tasks that cannot
+be done post-upgrade.
 
-- Click the ``Edit`` button in the box heading to show more options
-- Select VPC ``vpc-0af***********972``
-- Select ``Subnet`` to be ``us-east-1c``
-- Switch ``Auto-assign IPv6 IP`` to ``Enable``
-- Switch to ``Select existing security group`` and pick one of
+Avoid conducting the pre-upgrade too far in advance of the actual upgrade.
+Ideally, perform this phase a couple of hours or a day before.
 
-    - ``copr-frontend-sg``
-    - ``copr-backend-sg``
-    - ``copr-distgit-sg``
-    - ``copr-keygen-sg``
-    - ``copr-pulp-sg``
 
+Preparation
+-----------
 
-7. Configure storage
-....................
+Ensure you have the `helper playbook repository`_ cloned locally and navigate to
+the clone directory.
 
-- Click the ``Advanced`` button in the box heading to show more options
-- Update the ``Size (GiB)`` of the root partition
+Review the ``dev.yml``, ``prod.yml``, and ``all.yml`` configurations in the
+``./group_vars`` directory.  Pay particular attention to the ``old_instance_id``,
+``old_network_id``, and data volume IDs as **these MUST match the EC2 reality**.
 
-+----------------+-------------+-------------+
-|                | Dev         | Production  |
-+================+=============+=============+
-| **frontend**   | 50G         | 50G         |
-+----------------+-------------+-------------+
-| **backend**    | 20G         | 100G        |
-+----------------+-------------+-------------+
-| **keygen**     | 10G         | 20G         |
-+----------------+-------------+-------------+
-| **distgit**    | 20G         | 80G         |
-+----------------+-------------+-------------+
-| **pulp**       | 20G         | TODO        |
-+----------------+-------------+-------------+
+In the following moments, you will run following playbooks on your machine::
 
-- Turn on the ``Encrypted`` option
-- Select ``KMS key`` to whatever is ``(default)``
+    play-vm-migration-01-new-box.yml
+    play-vm-migration-02-migrate-backend-box.yml
+    play-vm-migration-02-migrate-non-backend-box.yml
+    play-vm-migration-03-rename-instances.yml
```

Comment:
I appreciate listing all the playbooks but I'd probably remove them and say something like "you will run several playbooks on your machine. During execution, explicitly specify ..."

Without reading ahead, I would proceed by executing all 4 playbooks for all instances. That would be bad.

Reply:
Done.